crates/astria-sequencer-relayer/tests/blackbox/helper.rs

```diff
@@ -106,6 +173,12 @@ pub struct TestSequencerRelayer {
 }
 
 impl TestSequencerRelayer {
+    pub fn shutdown(&self) {
```

Comment:
I think the aborts on the tasks are implied when they are droppoed.

Did you impl this because the server misbehaved otherwise?

Because this is a pretty harsh "shutdown" (more like "plug power") anyway, I wonder if this could just be moved into a drop impl so you don't have to explicitly `TestSequencerRelayer::shutdown` at the end of a test:

```rust
impl Drop for TestSequencerRelayer {
    fn drop(&mut self) {
        self.sequencer_relayer.abort();
        self.sequencer.abort();
        self.celestia.server_handle.stop().unwrap();
    }
}
```